packages/@ourworldindata/grapher/src/captionedChart/CaptionedChart.tsx

```diff
@@ -361,6 +372,15 @@ export class CaptionedChart extends React.Component<CaptionedChartProps> {
             height: chartHeight,
         }
 
+        const globeSwitcher: React.CSSProperties = {
```

Comment:
Again I couldn't see this used. Not familiar with the repo really, so of course it's likely me, [but I did search](https://github.com/search?q=repo%3Aowid%2Fowid-grapher%20globeSwitcher&type=code).